lib/empty_column.rb

```diff
@@ -1,5 +1,5 @@
 class EmptyColumn < Column
-  def initialize
-    super({"lists" => [], "cards" => []}, nil)
+  def initialize(settings = nil)
```

Comment:
If no settings is passed the default ```nil``` will be used. Next line will call the ```Column``` initializer with nil settings which in turn will call the ```Card``` initializer with nil settings. Then [this line](https://github.com/openSUSE/trollolo/pull/47/files#diff-da53d18413bddf22823980373cf4fff2R82) will break (Haven't tried it though).

I think it will be better if we used the same pattern regarding ```settings``` argument in all method signatures. Either use a default value everywhere or not. I think you can easily add a default value of ```nil``` in all methods by modifying the above line as such:

```
  @settings && @settings.no_task_checklists.include?(checklist.name)
```

Reply:
The linked line seem to have changed (or never made sense ?).  Maybe I have meant [this line](https://github.com/openSUSE/trollolo/pull/47/files#diff-da53d18413bddf22823980373cf4fff2R66) which resembles my proposal.

Reply:
I changed that line like you suggested. It's just that I rebased afterwards. So you can't see the old state anymore.